public/locales/pt/translation.json

```
@@ -4,6 +4,38 @@
   "noDevicesConnectedToSerial": "Nenhum dispositivo conectado a este serial",
   "editVisibleDevices": "Editar dispositivos visíveis",
   "createdat": "criado em",
+  "loadingInformation": "Carregando informações...",
+  "noReportsAvailable": "Nenhum relatório disponível",
+  "noDailyReportsAvailable": "Nenhum relatório diário disponível",
+  "NoLatency": "Sem informação de latência",
+  "NoLoss": "Sem informação de perda",
+  "Bad": "Ruim",
+  "Average": "Médio",
+  "AverageFemNoun": "Média",
+  "GoodFemNoun": "Boa",
+  "High": "Alto",
+  "lastDailyReports": "Últimos relatórios diários",
+  "cpeReboot": "Reinício do equipamento: {{-count}}",
+  "wanReboot": "Reinício da WAN: {{-count}}",
+  "minValueOfTheDay": "Valor mínimo do dia",
+  "maxValueOfTheDay": "Valor máximo do dia",
+  "meanValueOfTheDay": "Valor médio do dia",
+  "time": "ves",
+  "times": "vezes",
```

`time` e `times` pode ficar ambíguo com noções de tempo (ex.: segundos); typo na forma singular
```suggestion
  "timeRepetition": "vez",
  "timesRepetition": "vezes",
```